experimental/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ All notable changes to experimental packages in this project will be documented
 * fix(instrumentation-grpc): always set grpc semcov status code attribute with numeric value [#3076](https://github.com/open-telemetry/opentelemetry-js/pull/3076) @blumamir
 * fix(instrumentation): only call `onRequire` for full matches on core modules with sub-paths [#3451](https://github.com/open-telemetry/opentelemetry-js/pull/3451) @mhassan1
 * fix(instrumentation): add back support for absolute paths via `require-in-the-middle` [#3457](https://github.com/open-telemetry/opentelemetry-js/pull/3457) @mhassan1
+* fix(prometheus-sanitization): replace repeated `_` with a single `_` [3470](https://github.com/open-telemetry/opentelemetry-js/pull/3470) @samimusallam
```

Comment:
I might have missed it but is the PR has a test for repeated '_'?

Reply:
I updated the sanitization test to contain 2 invalid characters in sequence which will result in repeated `_` to be tested

Reply:
Cool, got it now :)